### ISSUE TITLE
`FileContents` encoding problem on Windows

### ISSUE THREAD

**Issue by juba** [manzt/anywidget#134]
Hi,

I recently encountered an issue with my pyobsplot widget on Windows (see https://github.com/juba/pyobsplot/issues/17). It seems that when the `_esm` or `_css` elements of a widget contain some special characters, Windows tries to read them with the default cp1252 encoding and fails, generating an UnicodeDecodeError. The problem doesn't appear on Linux (and I guess on MacOS).

After fiddling a bit, it seems that adding an `encoding` argument to `read_text()` in `FileContents.__str__` solves the issue:

```python
def __str__(self) -> str:
    if self._contents is None:
        self._contents = self._path.read_text(encoding="utf8")
    return self._contents
```

I'm really not sure it is a correct way to fix this, but if you think it is I can do a proper PR if you like.

Thanks !

**Comment by tlambert03** [comment]
I think that's the correct fix 👍

**Comment by manzt** [comment]
Thanks! Released in v0.4.3.
